app/hotels/src/allHotels/AllHotelsSearch.js

```diff
@@ -88,13 +85,7 @@ export class AllHotelsSearch extends React.Component<Props> {
         <FilterStripe />
         {isLoading && <FullPageLoading />}
         {!(isLoading || this.getCityIdFromData(data)) && (
-          <CenteredView>
-            <Text>
-              {location.length >= MIN_LOCATION_LENGTH
-                ? 'No relevant city was found.'
-                : 'Enter city name of your destination.'}
-            </Text>
-          </CenteredView>
+          <GeneralError errorMessage="No relevant city was found." />
```

Comment:
Even though this component seems stupid now - it's not going to be. We just don't have a graphic design for the error.